custom_news_feed/custom_news_feed.py

```diff
@@ -317,19 +347,54 @@ def checkPublishingDate(self, startdate, enddate):
         return True
 
 
+    def check_hashfile(self, newsident) -> bool:
+        """ Checks the existence of a hash file related to the news entry """
+        path = abspath(join(QgsApplication.qgisSettingsDirPath(), 'customnewsfeed'))
+        filename = abspath(join(path, newsident))              
+        if os.path.exists(filename):
+            return True
+        else:
+            return False
+
+    def delete_hashfile(self, newsident):
+        """ Deletes a hash file related to the news entry """
+        path = abspath(join(QgsApplication.qgisSettingsDirPath(), 'customnewsfeed'))
```

Comment:
REQ: Diesen Pfad in eine übergeordnete Variable packen, da es mindestens drei Wiederholungen davon gibt.